audio/public/js/audio_edit.js

```diff
@@ -0,0 +1,82 @@
+function AudioBlockStudio(runtime, element) {
+    $(element).find('.save-button').click(function(event) {
+        event.preventDefault();
+        $(element).find('form').submit();
+    });
+
+    $(element).find('form').submit(function(event) {
+        event.preventDefault();
+        var data = new FormData($(this).get(0));
+        var description = $(element).find('#description').val();
+        data.append('description', description);
+        runtime.notify('save', {state: 'start'});
+
+        $.ajax({
+            url: runtime.handlerUrl(element, 'studio_submit'),
+            type: 'POST',
+            data: data,
+            processData: false,
+            contentType: false,
+            success: function(response) {
+                runtime.notify('save', {state: 'end'});
+                window.location.reload();
+            }
+        });
+    });
+
+    $(element).find('.cancel-button').bind('click', function () {
+        if ('notify' in runtime) { //xblock workbench runtime does not have `notify` method
+            runtime.notify('cancel', {});
+        }
+    });
+
+    var sourcesField = document.getElementById('sources');
+    var embedUrlField = document.getElementById('embed_url');
+    var startTimeCheckbox = document.getElementById('start_time_checkbox');
+    var timeFields = document.getElementById('time_fields');
+
+    // Remove the existing openTab function and add the showContent function
+    window.showContent = function(contentType) {
+        var audioTab = document.getElementById('audioTab');
+        var podcastTab = document.getElementById('podcastTab');
+
+        if (contentType === 'audioTab') {
+            audioTab.style.display = 'block';
+            podcastTab.style.display = 'none';
+        } else if (contentType === 'podcastTab') {
+            podcastTab.style.display = 'block';
+            audioTab.style.display = 'none';
+        }
+    }
+
+    // Initialize the "Audio files" content by default
+    showContent('audioTab');
+    document.getElementById("audioRadio").checked = true;
```

Comment:
In case a podcast URL is already configured, shouldn't  we select its tab instead of the audio tab?

Reply:
Good point. Will check with Ali about what the UX should be for this case.